app/config/config.yml

```diff
@@ -7,6 +7,7 @@ imports:
 # http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
     locale: en
+    assets_version: v=1
```

Comment:
Add version of assets to config. This will help us to flush client-side cache for assets